http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -1073,9 +1082,11 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
             nodepoolQuery.updateStatus(nodepoolId, NodepoolStatus.Error).transaction
           )
         } yield ()
-      case _ =>
-        F.raiseError(
-          new RuntimeException(s"handleKubernetesError should not be used with a non kubernetes error. Error: ${e}")
+      case e =>
+        ev.ask.flatMap(ctx =>
```

Comment:
I don't think we should be throwing an exception within the top-level error handling function... there's not much we can do, as this essentially indicates we failed error handling at some point and an unintended exception bubbled up. 

Currently, this causes leonardo to crash if the block is hit.

Reply:
👍  if this handler is only used at top level

Reply:
👍 , this is better than crashing the Leo app I think.